src/entity/player.js

```diff
@@ -47,19 +57,34 @@ class Player extends Entity  {
         return sprite;
     }
 
+    damage() {
+        console.log("itai itai itai");
+    }
+
     // Engine level API
     load(board) {
         this.board = board;
+        this.howl = G_HOWL;
+        this.shootfx = new this.howl({
+            src: ["assets/audio/effects/Moonshot.Sfx.Graze.wav"],
+            loop: false
+        });
     }
 
     update(delta) {
+        // Elapsed serves as our counter to limit bullet firing frequency
         this.elapsed += delta;
         // NESW movement
-        let posBuf = this.movement(this.elapsed, this.container.position, undefined, undefined, undefined);
+        let posBuf = this.movement(undefined, this.container.position, undefined, undefined, undefined);
         this.container.position = posBuf;
         // Rotate towards mouse location
         // Convert mouse target point to game field coordinate system
         this.lookTowards(this.parent.toLocal(G_PIXI_APP.renderer.plugins.interaction.mouse.global));
+        // Hit testing
+        // Shooting
+        if(this.keysPressed["space"]) {
```

Comment:
Did we want to change to this RMB?

Reply:
Hah I thought per Discord discussion we wanted to leave this and related changes for later.